### ISSUE TITLE
Cognito no longer firing newPasswordRequired for AdminCreated users (on first log in)

### ISSUE THREAD

**Issue by hagen** [aws/amazon-cognito-identity-js#489]
Previously, on cognitoUser.authenticateUser where the Cognito User had just been created by Admin (i.e. in FORCE_PASSWORD_CHANGE, but enabled), the callback result would fire `newPasswordRequired`. This is no longer happening. The callback immediately fires `onSuccess`, with all three tokens available.
Users created by an Admin are no longer prompted for a new password, and are also unable to perform forgot passwords, or any other account activities while they remain in FORCE_PASSWORD_CHANGE. Anyone else seeing this? My endpoint is in ap-northeast-1

**Comment by itrestian** [comment]
Thanks for reporting this. Can you email me the user pool ID for this? Also can you give more details about your setup? How are you creating the users (using the CLI, the console etc). Do you have any lambda functions to confirm users, verify email, verify phone etc? Is this a newly created user pool? Are you using aliases?

**Comment by hagen** [comment]
Have PMd you :)

**Comment by itrestian** [comment]
Thanks! :+1:

**Comment by martinv13** [comment]
Same problem here, "Use case 23" of the README is not working for users created from the console on a fresh cognito pool (no lambda functions, etc.). `authenticateUser` throw an error if `newPasswordRequired` key is missing but does not seem to call the function when it is provided, which is strange...
Do you have an update on this?
Thanks!

**Comment by itrestian** [comment]
I'm not entirely sure this is the same problem. @hagen mentions that he is getting tokens immediately without firing the new password challenge while you say the function is not called. Can you check if the newPasswordRequired is indeed a function in your case?

**Comment by martinv13** [comment]
You are right I didn't manage to reproduce the issue I thought I had. Everything works fine.
Thank you

**Comment by iksnae** [comment]
is this still an issue? is there a workaround?

**Comment by pgerlich** [comment]
I am also seeing this issue.
Attempted several times with one user. After 4 attempts the callback stopped firing. No other (onSuccess/onFailure) callbacks fire, either.
Created a new user, attempted to use that account. No callbacks fire.
I did all of this in regular chrome and incognito mode with the same results.

Will reply if I find a solution.

**Comment by pgerlich** [comment]
What symptoms are you experiencing @iksnae ?

**Comment by buccfer** [comment]
I have the same problem. When I create users from the console, they are never prompted to enter a new password. It was working some time ago..
